lib/gltfPipeline.js

```diff
@@ -79,21 +79,31 @@ function processJSONWithExtras(gltfWithExtras, options) {
     convertDagToTree(gltfWithExtras);
     combineNodes(gltfWithExtras);
     combineMeshes(gltfWithExtras);
+    mergeDuplicateAccessors(gltfWithExtras);
+    removeDuplicatePrimitives(gltfWithExtras);
+    combinePrimitives(gltfWithExtras);
+    combineMeshes(gltfWithExtras);
+    removeUnused(gltfWithExtras, stats);
     generateNormals(gltfWithExtras);
-    // TODO: Combine primitives can be uncommented and added back into the pipeline once it is fixed, but right now there are too many issues with it to allow in the main pipeline.
-    // combinePrimitives(gltfWithExtras);
-    // Merging duplicate vertices again to prevent repeat data in newly combined primitives
-    // mergeDuplicateVertices(gltfWithExtras);
+    mergeDuplicateVertices(gltfWithExtras);
```

Comment:
As we talked about offline it may be worth testing if we still need this here, or if the final `mergeDuplicateVertices` is good enough.